app/lib/common/widget/FaqListItem.dart

```diff
@@ -0,0 +1,319 @@
+import 'package:effektio/screens/faq/Item.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+
+class FaqListItem extends StatefulWidget {
+  const FaqListItem({Key? key, required this.client, required this.faq})
+      : super(key: key);
+  final Client client;
+  final Faq faq;
+
+  @override
+  FaqListItemState createState() => FaqListItemState();
+}
+
+class FaqListItemState extends State<FaqListItem> {
+  static const TextStyle optionStyle =
+      TextStyle(fontSize: 30, color: Colors.white);
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+        onTap: () {
+          Navigator.push(
+            context,
+            MaterialPageRoute(
+              builder: (context) {
+                return FaqItemScreen(client: widget.client, faq: widget.faq);
+              },
+            ),
+          );
+        },
+        child: Column(
+          children: <Widget>[
+            // ListTile(
+            // title: Text(
+            //   widget.faq.title(),
+            //   style: optionStyle,
+            //   maxLines: 2,
+            // ),
+            // trailing: Icon(Icons.turned_in_outlined, color: Colors.grey),
+            // subtitle:
+            Column(
+              children: <Widget>[
+                Container(
+                  child: Card(
+                    child: Padding(
+                      padding: EdgeInsets.fromLTRB(8.0, 8.0, 8.0, 8.0),
+                      child: Column(
+                        children: [
+                          Padding(
+                            padding: EdgeInsets.fromLTRB(12.0, 4.0, 12.0, 4.0),
+                            child: Row(
+                              children: [
+                                Flexible(
+                                  child: Text(
+                                    widget.faq.title(),
+                                    style: GoogleFonts.montserrat(
+                                      color: Colors.white,
+                                      fontSize: 16,
+                                      fontWeight: FontWeight.w600,
+                                    ),
+                                  ),
+                                ),
+                                // new Spacer(),
+                                GestureDetector(
+                                  onTap: () => {
+                                    Fluttertoast.showToast(
+                                      msg: 'Bookmark icon tapped',
+                                      toastLength: Toast.LENGTH_SHORT,
+                                    ),
+                                  },
+                                  child: Image.asset(
+                                    'assets/images/bookmark.png',
+                                  ),
+                                )
+                              ],
+                            ),
+                          ),
+                          Padding(
```

Comment:
This entire element is a shortcut to display a "team"/group, you belong to. It should be its own component. Please refactor this and make this into a resuable component.